packages/babel-helpers/src/helpers/checkInRHS.js

```diff
@@ -0,0 +1,9 @@
+/* @minVersion 7.20.1 */
+
+export default function _checkInRHS(value) {
+  var type = typeof value;
+  if (type !== "object" && type !== "function") {
```

Comment:
```suggestion
  if (Object(value) !== value) {
```

This is a bit shorter.